code-challenge/README.md

```diff
@@ -31,7 +31,9 @@ _Hint_: There are a number of pattern examples in the Wikipedia article cited ab
 
 ### The Project
 
-You will be submitting this project on CodeSandbox. The `sandbox.config.json` file tells CodeSandbox to spin up a Node.js container for you to write your code and run your tests. The project is already configured to use [Babel](https://babeljs.io/), [ESLint](https://eslint.org/), and [Prettier](https://prettier.io/), so you can use the latest and greatest JavaScript features without wasting valuable setup time, or worrying about matters of style. Use one of the provided `test:` commands in `package.json` to run your tests - they'll automatically re-run when you edit them. Whenever you're ready, click on the link below, and then click the "Fork" button to get started. When you're finished, simply copy the sandbox URL from the browser, and email it back to us.
+You will be submitting this project on CodeSandbox. The `sandbox.config.json` file tells CodeSandbox to spin up a Node.js container for you to write your code and run your tests. Note that the project is only a server - you'll likely see a 502: Bad Gateway error in CodeSandbox's browser window, that's ok - ignore it and use the built-in terminal when needed. The project is already configured to use [Babel](https://babeljs.io/), [ESLint](https://eslint.org/), and [Prettier](https://prettier.io/), so you can use the latest and greatest JavaScript features without wasting valuable setup time, or worrying about matters of style. Use one of the provided `test:` commands in `package.json` to run your tests - they'll automatically re-run when you edit them. Whenever you're ready, click on the link below, and then click the "Fork" button to get started. When you're finished, simply copy the sandbox URL from the browser, and email it back to us.
+
+Out of respect for your time, please spend no more than three hours on the project. If you run into any trouble with the assignment, CodeSandbox, or the time limitation, please contact us.
```

Comment:
I was going to suggest we keep something about a time limit in here, and here it is! I like it.